packages/adp-tooling/src/preview/routes-handler.ts

```diff
@@ -119,7 +120,7 @@ export default class RoutesHandler {
                 return;
             }
 
-            const fullPath = path.join(sourcePath, FolderTypes.CHANGES, FolderTypes.FRAGMENTS);
+            const fullPath = path.join(sourcePath, DirName.Changes, DirName.Fragment);
```

Comment:
`FolderTypes.FRAGMENTS` **plural** is changed to ` DirName.Fragment` **singular**
Is this expected?